src/main/java/org/sefglobal/scholarx/model/Comment.java

```diff
@@ -0,0 +1,43 @@
+package org.sefglobal.scholarx.model;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Table;
+import javax.persistence.ManyToOne;
+
+@Entity
+@Table(name = "comment")
+public class Comment extends BaseScholarxModel {
+    @ManyToOne
+    private Mentee mentee;
```

Comment:
I think we'll have to add the relationship to the Mentee model and Ignore the mentee comments with JsonIgnoreProperties. Does this work properly without adding that?

Reply:
yeah